### ISSUE TITLE
Cannot find name 'ErrorOptions' with version 2

### ISSUE THREAD

**Issue by lpellegr** [vaadin/router#897]
Upgrading from version 1 to version 2 in a Typescript project, upon compilation I get the following error:

> [tsc] node_modules/@vaadin/router/dist/resolver/resolver.d.ts(11,49): error TS2304: Cannot find name 'ErrorOptions'.

here is my tsconfig.json:

``` 
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": false,
    "experimentalDecorators": true,
    "importHelpers": true,
    "inlineSources": true,
    "lib": ["es2021", "dom", "dom.iterable"],
    "module": "es2020",
    "moduleResolution": "node",
    "noEmitOnError": true,
    "outDir": "out",
    "sourceMap": true,
    "target": "es2021",
    "strict": false,
    "useDefineForClassFields": false
  },
  "include": [
    "src/**/*.ts"
  ]
}
``` 

Typescript version _5.6.3_.

**Comment by lpellegr** [comment]
ErrorOptions requires es2022, is it expected?

**Comment by platosha** [comment]
Thanks for pointing out. The fix will be released in v2.0.1 soon.
